drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -94,6 +102,14 @@ static const clock_ip_name_t i2c_clk_root[] = {
 };
 #endif
 
+#if defined(CONFIG_DMA_NXP_SDMA)
+static const clock_ip_name_t sdma_clocks[] = {
+	kCLOCK_Sdma1,
+	kCLOCK_Sdma2,
+	kCLOCK_Sdma3,
+};
+#endif /* CONFIG_DMA_NXP_SDMA */
```

Comment:
There is a macro `SDMA_CLOCKS` in nxp hal for this. Just to make you aware. I think it is better not to use it for clarity.